### PR TITLE
JDF-883 - Add Fuse 7.10.1 in stacks

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

**Comment by apupier** [comment]
the error doesn't sound related to my PR:

```
Error:      Non-parseable POM /home/runner/.m2/repository/org/jboss/bom/eap/jboss-javaee-6.0-with-hibernate/6.2.0.GA/jboss-javaee-6.0-with-hibernate-6.2.0.GA.pom: entity reference names can not start with character '&' (position: START_TAG seen ...text/javascript">\n    if (!/\/logout.*/.test(location.pathname) &&... @64:49)  @ /home/runner/.m2/repository/org/jboss/bom/eap/jboss-javaee-6.0-with-hibernate/6.2.0.GA/jboss-javaee-6.0-with-hibernate-6.2.0.GA.pom, line 64, column 49 -> [Help 2]
```

**Comment by lhein** [comment]
@jeffmaury any idea?

File: stacks.yaml [hunk 1]
```diff
@@ -5382,6 +5382,26 @@ availableRuntimes:
         runtime-md5: c51744a5e35578a06a9e6499fbfe2348,
         wtp-runtime-type: org.fusesource.ide.fuseesb.runtime.70
    }
+   
+      ########################
+   #  Red Hat Fuse 7.10.0 #
```
Comment (lhein) [inline stacks.yaml]: shouldn't it be 7.10.1?

**Comment by apupier** [comment]
build error reproduced on main branch. Reported blocker issue https://issues.redhat.com/browse/JDF-884

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**